tests/settings.py

```diff
@@ -21,7 +21,7 @@
 MIDDLEWARE_CLASSES = (
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.middleware.common.CommonMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
+    # 'django.middleware.csrf.CsrfViewMiddleware',
```

Comment:
I was just thinking that if it's going to be perpetually commented-out, why not remove it entirely?